storage/country.cpp

```diff
@@ -3,13 +3,16 @@
 #include "platform/mwm_version.hpp"
 #include "platform/platform.hpp"
 
+#include "coding/reader.hpp"
+
 #include "base/logging.hpp"
 #include "base/stl_helpers.hpp"
 
 #include "3party/jansson/myjansson.hpp"
 
-#include "std/utility.hpp"
+#include <utility>
 
+using namespace std;
```

Comment:
If you put `using namespace std;` here let's remove `std::` from the file. As far as I see there's some `std::` in Storage::OnMapFileDownloadFinished().

Reply:
Oh! Sorry.